cypress/integration/page.spec.js

```diff
@@ -1,7 +1,132 @@
-// WRITE TESTS HERE
+/// <reference types="cypress" />
```

Comment:
Ok, but autocomplete should work without this line

Reply:
ok